### PR TITLE
Bookstore: connect to API

### PR DESCRIPTION
**Pull request by bill7pearl**
# In this PR I have:
- Made use of the Bookstore API "https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/SukL6UM81zzFEr8SdMgf/books" as my backend.
- Used fetch with createAsyncThunk to make API calls.
- Fetched my books from Bookstore with useEffect.
- Refactored my add book and remove book features to persist my changes in the server.
- Modified my reducer logic to include HTTP requests to the Bookstore API.

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @bill7pearl , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/homePage.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import Form from './Form';
-import BookRender from './BookRender';
+import BookRenderer from './bookRender';
```
Comment (tugsem) [inline src/components/homePage.js]: - [x] Good job connecting your app to API! 👍 Your app is working properly when I fix this small issue: Kindly fix the typo here, your file name doesn't match: 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hi @bill7pearl , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️Proffessional documentation
✔️Descriptive commit messages, branch name
✔️Linters pass
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/homePage.js (hunk 1 above)
Comment (bill7pearl) [reply src/components/homePage.js]: worked fine in my end, but I've made some changes hopefully it will work now

**Review by danmainah — APPROVED** [review]
Hi @bill7pearl , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**